kratos/tests/cpp_tests/utilities/test_smp_storage.cpp

```diff
@@ -0,0 +1,57 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Suneth Warnakulasuriya
+//
+
+// System includes
+#include <vector>
+
+// Project includes
+#include "utilities/reduction_utilities.h"
+#include "utilities/parallel_utilities.h"
+#include "utilities/smp_storage.h"
+#include "testing/testing.h"
+
+namespace Kratos {
+namespace Testing {
```

Comment:
```suggestion
namespace Kratos::Testing {
```

welcome to the future :)